packages/CalendarCarousel/index.tsx

```diff
@@ -265,8 +333,14 @@ function CalendarCarousel<T>({
 
     const calendarDays = [...prevDays, ...days, ...nextDays];
 
+    for (let i = 0; i <= markedDayEvents.length - 1; i++) {
+      if (markedDayEvents[i].day === eventSwitch) {
+        const getEventDay = markedDayEvents[i].events; dayEvents.push(getEventDay);
+      }
+    }
```

Comment:
In this case I think using [reduce](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) would be more ideal.